roles/openafs_client/vars/CentOS-8.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# OpenAFS installation paths.
+afs_viceetcdir: "/usr/vice/etc"
+afs_afscachedir: "/usr/vice/cache"
+
+# Service name
+afs_client_service: openafs-client
+
+# OS dependent variables
+afs_firewall: firewalld
+
+# Install dkms by default in CentOS 8.
+_afs_client_install_dkms: yes
+
+# Build default options.
+_afs_client_configure_opts: >
```

Comment:
See the note attached to roles/openafs_server/vars/CentosOS-8.yaml in "Default selinux mode to permissive on CentOS 8"

Reply:
should be fixed with the openafs_devel change to add CentOS-8 there?